src/Data/Container/Indexed.agda

```diff
@@ -104,7 +104,7 @@ module _ {i₁ i₂ o₁ o₂}
 
 -- Degenerate cases where no reindexing is performed.
 
-module _ {i o c r} {I : Set i} {O : Set o} where
+module _ {i o c r : _} {I : Set i} {O : Set o} where
 
```

Comment:
This is surprising. Does this work because adding `: _` means they *all* share the same
metavariable which gets solved because of the `I : Set i` type annotation which
implies `i : Level`?